recipes/ios/src/ios_browser.m

```diff
@@ -11,3 +11,30 @@ void ios_open_url(char *url)
 	NSString *nsurl = [NSString stringWithCString:(char *)url encoding:NSUTF8StringEncoding];
 	[[UIApplication sharedApplication] openURL:[NSURL URLWithString: nsurl]];
 }
+
+/*
+ * Webview support
+ */
+void load_url_webview(char *url)
+{
+    NSString *nsurl = [NSString stringWithCString:(char *)url encoding:NSUTF8StringEncoding];
+    UIWebView *webView = [[UIWebView alloc] initWithFrame: CGRectMake(0, 0, 320, 480)];
+    UIWindow *window = [[UIApplication sharedApplication] keyWindow];
+    UIView *view = [window.rootViewController view];
+    [view addSubview:webView];
+    NSURL *ur = [[NSURL alloc] initWithString: nsurl];
+    NSURLRequest *req = [[NSURLRequest alloc] initWithURL: ur];
+    [webView loadRequest: req];
+    [req release];
+    [ur release];
+  
+    UIButton *button = [UIButton buttonWithType:UIButtonTypeRoundedRect];
+    [button setTitleColor:[UIColor blackColor] forState:UIControlStateNormal];
+    [button setTitle:@"X" forState:UIControlStateNormal];
```

Comment:
Would be better to use the back button interface like all the rest of the iOS apps.